pkg/controller/controller_test.go

```diff
@@ -89,6 +89,7 @@ const (
 	testServiceInstanceCredentialSecretName = "test-secret"
 	testOperation                           = "test-operation"
 	testClusterID                           = "test-cluster-id"
+	testOSBTimeOut                          = 80 * time.Second
```

Comment:
it's used only in one place, please move that directly to `TestReconcileClusterServiceBrokerSetOSBTimeOut`